.pipelines/ci.yml

```diff
@@ -3,6 +3,7 @@ trigger:
   branches:
     include:
     - master
+    - refs/tags/*
```

Comment:
AZP does not support better filtering. Only globbing at this point. Thus fire for all tags.

Reply:
Can you specify the tag format like this? https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#tags

Reply:
I feel like an idiot. I have completely missed this line. I found the solution and did not scroll down further.

After an experiment on my repo I confirm it works as prefix globing. No regex, but at least filtering `v2*` gives better refinement.

Thank you for catching this.